src/MessagingInput/MessagingInput.tsx

```diff
@@ -54,13 +55,14 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
     checkbox,
     className,
     disableSendButton,
-    labelText = "Send a message",
+    label,
     newlineOnEnter,
     onBlur,
     onChange,
     onFocus,
     onReplyCancel,
     onSubmit,
+    placeholder = "Send a message",
```

Comment:
A placeholder inside the input is now the default, rather than a label outside.

_Before_

<img width="930" alt="before-1" src="https://user-images.githubusercontent.com/12616928/126706155-ffbda258-4031-43ce-b9a7-2d088b7d9f7f.png">

_After_

<img width="940" alt="after-1" src="https://user-images.githubusercontent.com/12616928/126706153-fbf38818-d67e-4e67-86be-a7693309b99b.png">

-----

That said, we still need to support labels outside for a few edge cases, e.g. this input in the upcoming welcome flow.

![welcome](https://user-images.githubusercontent.com/12616928/126706268-1e305e11-3c36-4f7f-afb0-5db72a783390.png)